### PR TITLE
Upload ONLY Assignment4

### PR DESCRIPTION
**Pull request by ramirolc02**

### CODE REVIEW

File: Assignment4/ramirolc02-200356/Task07.py [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+g.namespace_manager.bind('vcard', Namespace("http://www.w3.org/2001/vcard-rdf/3.0#"), override=False)
+g.parse(github_storage+"/rdf/example6.rdf", format="xml")
+
+# %% [markdown]
+# **TASK 7.1: List all subclasses of "LivingThing" with RDFLib and SPARQL**
+
+# %%
+
+# RDFlib
+for s,p,o in g.triples((None, RDFS.subClassOf, None)):
```
Comment (anaigmo) [inline Assignment4/ramirolc02-200356/Task07.py]: subclass of :LivingThing

**Comment by anaigmo** [comment]
Task6 is fine, but 7 needs re-work:
* In rdflif alternatives of tasks 7.2 and 7.3, you output the results separately (e.g. one output for instances of person, another for instances of animal), they should be unified
*in task 7.4, output the name, not the IRI
* In 7.5, you are querying entities that know entities that know other entities, but the task is asking for the entities that know at least 2 entities (same level, not one following the other)

**Comment by anaigmo** [comment]
Thanks for the corrections! But you still haven't fixed query 7.5, check my comment above

**Comment by ramirolc02** [comment]
Done now, I thought it was 2 levels of entities, not 2 entities on same level. Thanks for clarification.

**Pull request merged**